config/checker/io.vavr.astub

```diff
@@ -85,3 +85,8 @@ interface Option<T> {
 
   static <T> io.vavr.control.Option<@NonNull T> of(@Nullable T value);
 }
+
+
+interface Try<T> {
+  static <T> Try<T> of(CheckedFunction0<? extends @Nullable T> supplier);
```

Comment:
Theoretically `static <T> Try<@PolyNull T> of(CheckedFunction0<? extends @PolyNull T> supplier);` could work... pls try, although no guarantees coz https://github.com/typetools/checker-framework/issues/979

Reply:
This will effectively make `of` behave as if it was "overloaded" with the two following versions:

* `static <T> Try<@Nullable T> of(CheckedFunction0<? extends @Nullable T> supplier);`
* `static <T> Try<@NonNull T> of(CheckedFunction0<? extends @NonNull T> supplier);`

Reply:
Unfortunately it seems not working :/

Reply:
;(